unfoldingWord/translationCore
# Feature/mc 732 814

#### This pull request addresses:

- This PR adds the latest version of scripture pane to tC.
- removed padding on the left of ModuleWrapperContainer from app.js
- fixed bug where developer mode card would not show up
- Added latest commit head for scripture pane

#### How to test this pull request:
- Try switching to developer mode and now you will see the ToolsTester Card in the `Load Tool` sub-tab.
- you should now see the latest version of scripture pane. (if not make sure you run 'npm run pull-apps ')


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/853)
